app/feats/insurance_claim_feat.py

```
@@ -1,175 +1,561 @@
-        status=claim.status.value,
-        message=f"{claim_type.replace('_', ' ').title()} claim submitted.",
+
+
+@feat_shell("FEAT-STOR-003")
+def _submit_insurance_claim_impl(
+    *,
+    canonical_context: CanonicalContext,
+    entitlement_id: str,
+    claim_subject: dict,
+    correlation_id: str | None = None,
+    idempotency_key: str | None = None,
```

Ruff (0.16.0)

[warning] Unused function argument: `idempotency_key`

(ARG001)